cli/fathom_web/commands/serve.py

```diff
@@ -1,6 +1,7 @@
-import contextlib
+from contextlib import contextmanager
```

Comment:
Does this deep importing actually import only that submodule instead of the entire module?

Reply:
contextmanager is a function rather than a module. The same code runs in any case: all the top-level code in the contextlib module runs, from top to bottom. The only difference is now only the contextmanager symbol ends up in our namespace, not contextlib. I favor deep importing in general because wordspace is big. I don't need to call you Bianca Danforth every time I refer to you, so I figure I shouldn't need to say contextlib.contextmanager either. :-)